SemiRP/Utils/Chat.cs

```diff
@@ -38,60 +36,75 @@ public static void SendGradientRangedChat(Player from, float radius, Color baseC
                 }
             }
         }
+        
+        public static void ClientChat(Player player, string message)
+        {
+            player.SendClientMessage(Color.White, message);
+        }
 
         public static void SendMeChat(Player from, string message)
         {
-            var text = "* " + from.Name + " " + message;
+            var text = from.Name + " " + message;
             SendRangedChat(from, SemiRP.Constants.PROXIMITY_RADIUS, Constants.Chat.ME, text);
             from.SetChatBubble(text, Constants.Chat.ME, Constants.Chat.CHAT_BUBBLE_ME_RANGE, (int)(text.Length / Constants.Chat.CHAT_BUBBLE_TIME_FACTOR));
         }
 
         public static void SendDoChat(Player from, string message)
         {
-            SendRangedChat(from, SemiRP.Constants.PROXIMITY_RADIUS, Constants.Chat.ME, "* " + message + " ((" + from.Name + "))");
+            SendRangedChat(from, SemiRP.Constants.PROXIMITY_RADIUS, Constants.Chat.ME, message + " ((" + from.Name + "))");
         }
+
         public static void ErrorChat(Player player, string message)
         {
-            player.SendClientMessage(Color.White, "[" + Constants.Chat.ERROR_TAG + "ERREUR" + Color.White + "] "+message);
+            player.SendClientMessage(Constants.Chat.ERROR, message);
```

Comment:
No more "ERROR" tag in the chat ?

Reply:
Effectively. It is redundant to have a full chat with line-by-line "tags".

This is not a problem from a user point of view:
- he knows his action didn't go as planned
- he always see the colored error